students/HarveyHerela/lesson09/mailroom_test.py

```diff
@@ -0,0 +1,117 @@
+import unittest
+
+import mailroom_actions
+import mailroom_db
+
+class DBTest(unittest.TestCase):
+
+    def create_default_db(self):
+        db = mailroom_db.DonorCollection()
+        # Some default data
+        d1 = mailroom_db.Donor("Jimmy", "Stewart")
+        d1.add_donation(50)
+        d2 = mailroom_db.Donor("Cary", "Grant")
+        d2.add_donation(30)
+        d2.add_donation(40)
+        d3 = mailroom_db.Donor("Audrey", "Hepburn")
+        d3.add_donation(80)
+        d3.add_donation(90)
+        d3.add_donation(100)
+        db.add_donor(d1)
+        db.add_donor(d2)
+        db.add_donor(d3)
+        return db
```

Comment:
You can use the `unittest.TestCase.setUp` method for shared test data like this

Reply:
I thought about that, but the impression I got was that setup runs every time, and I didn't want to run it everytime (namely, when testing adding a donor). I guess I could've created another subclass for testing that functionality?